Express/test/api.test.js

```diff
@@ -218,6 +221,32 @@ test('Test for valid number for filter', function (done) {
         });
 });
 
+test('Test for item/key query', function (done) {
+    request
+        .get(o12)
+        .end(function (err, res) {
+            if (err) return done(err);
+            checkResponse(res, 400);
```

Comment:
Should be 200 instead of 400, but it looks like `checkResponse()` actually doesn't do anything because your test passed anyway...

Reply:
Not quite sure about the hanging response thing, if the --detectOpenHandles flag works in command line you can add it here and the github tests will pass:
https://github.com/The-Data-Grid/The-Data-Grid/blob/6e3371073b33c1669d780c2fceee2b8b0d27e88b/.github/workflows/api.yml#L53